### PR TITLE
update bitcoin-xt and add bitcoin-classic

### PR DESCRIPTION
**Pull request by jefdaj**

### CODE REVIEW

File: pkgs/applications/altcoins/bitcoin-classic.nix [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  buildInputs = [ pkgconfig autoreconfHook openssl db48 boost zlib
+                  miniupnpc utillinux protobuf ]
+                  ++ optionals withGui [ qt4 qrencode ];
+
+  configureFlags = [ "--with-boost-libdir=${boost.lib}/lib" ]
+                     ++ optionals withGui [ "--with-gui=qt4" ];
+
+  meta = {
+    description = "We are hard forking bitcoin to a 2 MB blocksize limit. Please join us.";
```
Comment (jagajaga) [inline pkgs/applications/altcoins/bitcoin-classic.nix]: IMO that's not a description.

**Comment by jefdaj** [comment]
Think I should just copy the "peer-to-peer electronic cash system" one from bitcoin core each time? Or make up my own? Either is fine with me.

**Comment by vcunat** [comment]
Yes, I think that would be fine. The long description might attempt do describe the differences, but probably better not in this advertisement style.

**Comment by jefdaj** [comment]
Cool I'll edit it. I didn't mean for it to be in an advertisement style. I just always copy the descriptions verbatim when packaging things; didn't know you're supposed to fix them up.

**Comment by jefdaj** [comment]
So, good to go then? Anything else I should do?

**Comment by jagajaga** [comment]
Rebase commits, squash them into one and name it to follow our guide in `CONTRIBUTING.md`

**Comment by zimbatm** [comment]
@jefdaj do you need help with git shenanigans ?  Basically run `git rebase -i origin/master` and then you can squash all the commits into one. Then I would `git reset HEAD` and then commit the update and the new packages separately. `git add path/to/bitcoin-xt` and then `git commit -m "bitcoin-xt: 0.11A -> 0.11D"` and then add the rest and `git commit -m "bitcoin-classic: init at 0.11.2.cl1.b1"`

**Comment by vcunat** [comment]
Split and pushed.

**Comment by jefdaj** [comment]
Sorry I kind of abandoned this! Just got super busy. Finally got some free time and came back to fix it, but no need now. Thanks @vcunat!

**Pull request closed without merging**